workflow/scripts/prioritization/reference.py

```
@@ -61,12 +61,15 @@ def parse_exome(self, gtfFile):
 
 class Counts:
     def __init__(self, countFile):
+        print(countFile)
         # parse counts
         self.counts = {}
         if countFile is not None and countFile is not "":
```

Ruff (0.8.2)

Use `!=` to compare constant literals

Replace `is not` with `!=`

(F632)